client/src/components/Related/YourOutfit.jsx

```diff
@@ -1,43 +1,61 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
@sm-fm here is the functionality I added to YourOutfit - the commits before this are for my previous PR